x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/time_range_step/time_range_picker.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+import React, { Fragment, FC, useContext, useState, useEffect } from 'react';
+import { EuiDatePickerRange, EuiDatePicker } from '@elastic/eui';
+
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+
+import { KibanaContext, isKibanaContext } from '../../../../../data_frame/common/kibana_context';
+
+const euiSize = +theme.euiSize.replace('px', '');
+const WIDTH = `${euiSize * 32}px`;
```

Comment:
Although this is probably the 'correct' way of setting the width, I wonder if it would just be simpler to define this as a hardcoded value of `512px`?